code/modules/reagents/reagent_containers/food/cans.dm

```diff
@@ -178,28 +178,28 @@
 
 /obj/item/reagent_containers/food/drinks/cans/monster_red
 	name = "\"Code RED\" Claw Energy"
-	desc = "Claw energy, for when you need the focus of a hawk and the energy of a cheetah. A warning label says drinking to much may cause heart damage, liver damage, and not living anymore. \
+	desc = "Claw energy, for when you need the focus of a hawk and the energy of a cheetah. A warning label says drinking too much may cause heart damage, liver damage, and not living anymore. \
 	Red cans are drunk faster."
-	icon_state = "claw_blue"
+	icon_state = "claw_red"
```

Comment:
```suggestion
	icon_state = "claw_blue"
```

Reply:
Why are you giving them the WRONG sprites again?!?!!